controllers/configentry_controller.go

```diff
@@ -177,8 +177,14 @@ func (r *ConfigEntryController) ReconcileEntry(
 	return ctrl.Result{}, nil
 }
 
-func (r *ConfigEntryController) consulNamespace(kubeNS string) string {
-	return namespaces.ConsulNamespace(kubeNS, r.EnableConsulNamespaces, r.ConsulDestinationNamespace, r.EnableNSMirroring, r.NSMirroringPrefix)
+func (r *ConfigEntryController) consulNamespace(kubeNS string, isConsulNamespaced bool) string {
+	if isConsulNamespaced {
+		return namespaces.ConsulNamespace(kubeNS, r.EnableConsulNamespaces, r.ConsulDestinationNamespace, r.EnableNSMirroring, r.NSMirroringPrefix)
+	}
+	if r.EnableConsulNamespaces {
+		return common.DefaultConsulNamespace
+	}
+	return ""
```

Comment:
Ahh, because setting "default" fails in OSS right? Let's just return "" in both cases then since "" == "default" in consul?

Reply:
maybe not? https://app.circleci.com/pipelines/github/hashicorp/consul-k8s/1084/workflows/5b8f77bd-f048-436b-b043-7718155bc80f/jobs/4651

Reply:
Ahh, because we're using this same method when we create the namespace and obviously "" doesn't work for a namespace name. K leave it as it!